db/schema.rb

```diff
@@ -28,7 +28,7 @@
   end
 
   create_table "company_values", force: :cascade do |t|
-    t.string "title"
+    t.string "title", null: false
```

Comment:
Where is this change coming from? I can't find associated migration

Reply:
Ok, noted, sorry for that.